roles/openshift-applier/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-
+client: kubectl
```

Comment:
To stay backward compatible, this should be `oc`

Reply:
agreed. will change that.